changelog.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## NEXT
+
+- Added the `cai_causal_graph.time_series_graph.TimeSeriesGraph.get_nodes_at_lag` and
+  `cai_causal_graph.time_series_graph.TimeSeriesGraph.get_contemporaneous_nodes` methods to the
+  `cai_causal_graph.time_series_graph.TimeSeriesGraph` class to get the nodes at a given lag, respectively.
```

Comment:
"o get the nodes at a given lag, respectively." missing explanation of what second method does.